shared/reports/resources.py

```diff
@@ -93,18 +90,13 @@ def __init__(
             self._details = {}
             self._lines = []
 
-        # <line_modifier callable>
-        self._line_modifier = line_modifier
         self._ignore = _ignore_to_func(ignore) if ignore else None
 
-        if line_modifier:
-            self._totals = None  # need to reprocess these
+        # totals = <ReportTotals []>
```

Comment:
is this supposed to mean `totals: list[ReportTotals]` ?

It seems is more like `ReportTotals | list | None`, where `list` is the serialized version of `ReportTotals`

My point is the comment is a bit useless